about.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script
+      async
+      src="https://www.googletagmanager.com/gtag/js?id=G-0GGDWZYGDE"
+    ></script>
+    <script>
+      window.dataLayer = window.dataLayer || [];
+      function gtag() {
+        window.dataLayer.push(arguments);
+      }
+      gtag("js", new Date());
+
+      gtag("config", "XXXXX");
+    </script>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width" />
+    <title>Capstone</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+
+  <body class="about-body">
+    <div class="optionsheadercontainer">
+      <ul class="optionsheader">
+        <li><i><img class="icon1" src="./imgs/facebook-icon-png-774.png" alt="" href="#"></i></li>
+        <li><i><img class="icon2" src="./imgs/Twitter.png" alt="" href="#"></i></li>
+        <li class="optiontext1"><a href="#">English</a></li>
+        <li class="optiontext2"><a href="#">log in</a></li>
+      </ul>
+    </div>
+      <header class="header">
+          <div class="headerelement headerelement1"></div>
+          <nav class="headerelement headerelement2">
+            <div class="lines">
+              <div class="menuline line-1"></div>
+              <div class="menuline line-2"></div>
+              <div class="menuline line-3"></div>
+            </div>
+            <ul class="menuitems">
+              <li class="menuitem menuitem-1">
+                  <a href="./index.html">Home</a>
+              </li>
+              <li class="menuitem menuitem-2">
+                <a href="./about.html">About</a>
+              </li>
+              <li class="menuitem menuitem-3">
+                <a href="#">lorem</a>
+              </li>
+              <li class="menuitem menuitem-4">
+                <a href="#">Ipsum</a>
+              </li>
+            </ul>
```

Comment:
- [ ] Talking about the burger menu, the background color could have transparency, this will make a better design on your project, please work on it. 🐱 
![image](https://user-images.githubusercontent.com/20563252/128289394-51b838a8-d1a4-40f8-bdab-751a68a8d294.png)

Reply:
Ok that's a great idea, I added transparency for the hamburger menu 😄